website/src/community/integrations.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: section
+title: "Integrations"
+permalink: /community/integrations/
+section_menu: section-menu/community.html
+redirect_from:
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Integrations
+
+The following external projects integrate with Apache Beam:
+
+## Kettle
+
+Kettle is a tool for desinging data pipelines using a graphical user interface.
+The resulting pipelines can be translated to Beam pipelines.
+
+- [Kettle-Beam Integration](https://github.com/mattcasters/kettle-beam)
+- [Kettle-Beam Examples](https://github.com/mattcasters/kettle-beam-examples)
+
+- [Kettle Project page](https://community.hitachivantara.com/docs/DOC-1009855)
```

Comment:
Is this really the project page? The landing page if you search is the Hitachi Vantara product marketing page. I did not quite realize the nature of the linking when I suggested it. There's probably a bigger conversation to have about the Beam project / PMC appearing to promote particular organizations or products.

Reply:
Good point. Probably safer to just link the pages hosting the code.